packages/cli-lib/api/sandboxes-sync.js

```diff
@@ -0,0 +1,37 @@
+const http = require('../http');
+const SANDBOXES_SYNC_API_PATH = 'sandboxes-sync/v1';
+
+// fromHubId is the parent portal here, which is required as the first param. Parent account key can only run the sync
+async function initiateSync(fromHubId, toHubId, tasks, sandboxHubId) {
+  return http.post(fromHubId, {
+    body: {
+      command: 'SYNC',
+      fromHubId,
+      toHubId,
+      sandboxHubId,
+      tasks,
+    },
+    timeout: 60000,
+    uri: `${SANDBOXES_SYNC_API_PATH}/tasks/initiate/async`,
+  });
+}
+
+async function fetchTaskStatus(accountId, taskId) {
+  return http.get(accountId, {
+    uri: `${SANDBOXES_SYNC_API_PATH}/tasks/${taskId}`,
+  });
+}
+
+async function fetchTypes(accountId, toHubId) {
+  return http.get(accountId, {
+    uri: `${SANDBOXES_SYNC_API_PATH}/types${
```

Comment:
A little confused about what `types` represents here. Is this how we determine whether a portal is a sandbox account or a PROD account?

Reply:
This one is to fetch sync types, eg: object schemas, workflows, etc. We have types that are only able to be run once per portal, so this fetch is necessary to give us what we can/can't sync.